index.bs

```diff
@@ -2789,6 +2789,50 @@ the [=authenticator=] MUST:
     attStmtTemplate .within $$attStmtType
     ```
 
+### Signature Formats ### {#signature-attestation-types}
```

Comment:
Please make this title more specific clearly restrict its scope to only the kinds of signatures that it applies to.  Right now it reads like it applies to all signatures.

For instance, the new title could be "Signature Formats for Packed Attestations and Assertion Signatures" (if that's the correct scoping).

Reply:
Changed to added FIDO-U2F attestation format also.